tests/validation/tests/v3_api/test_cis_scan.py

```diff
@@ -32,32 +32,86 @@
         "not_applicable": 19, "total": 97, "fail": 0
     },
     "rke-cis-1.5": {
-        "permissive": {"pass": 58, "fail": 0, "skip": 14},
+        "permissive": {"pass": 60, "fail": 0, "skip": 12},
         "hardened": {"pass": 72, "fail": 0, "skip": 0},
         "not_applicable": 20, "total": 92, "fail": 0
     }
 }
 DEFAULT_TIMEOUT = 120
 cluster_detail = {
-    "cluster": None, "nodes": None, "name": None
+    "cluster_14": None, "nodes": None, "name": None, "cluster_15": None
 }
 
 
-def test_cis_scan_run_scan():
-    cluster = cluster_detail["cluster"]
+def test_cis_scan_run_scan_hardened_14():
+    cluster = cluster_detail["cluster_14"]
     scan_detail = run_scan(cluster, USER_TOKEN, "hardened")
     report_link = scan_detail["links"]["report"]
     verify_cis_scan_report(
         report_link, token=USER_TOKEN,
-        test_total=scan_results[CIS_SCAN_PROFILE]["total"],
-        tests_passed=scan_results[CIS_SCAN_PROFILE]["hardened"]["pass"],
-        tests_skipped=0
+        test_total=scan_results["rke-cis-1.4"]["total"],
```

Comment:
Can we take this opportunity to encapsulate the `scan_results` dictionary retrieval into a method say `get_scan_results` where we just pass the `cis_scan_profile` we're trying to get the results for?